nbs/010_engine.ipynb

```diff
@@ -40,7 +40,9 @@
     "from abc import ABC, abstractmethod\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    res = pd.DataFrame(columns=("X", "Y"), data=e.run_query(Relation(name='R',terms=[FreeVar(name='X'),FreeVar(name='Y')])))
</pre><p>since we instantiate all results using differential, we agreed to return to always returning dataframes from spannerflow, and not iterators.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spannerlib/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Yes, this the changes we made to the original tests before we agreed on this, will change</p><p> </p>

Reply:
<p>Changed</p><p> </p>